.github/workflows/cargo-update.yml

```
@@ -22,8 +22,8 @@ jobs:
         continue-on-error: true
         run: |
           git config user.name 'Phylum Bot'
-          git config user.email 'phylum-bot@users.noreply.github.com'
-          git commit -a -m "build: Bump `Cargo.lock` dependencies"
+          git config user.email '69485888+phylum-bot@users.noreply.github.com'
+          git commit -a -m "Bump dependencies"
```

Also considered:

```suggestion
          git commit -a -m 'Bump `Cargo.toml` dependencies'
```

But I decided simpler was better